src/mmw/js/src/modeling/tests.js

```diff
@@ -527,7 +527,7 @@ var modificationsSample1 = {
 
 var modificationsSample2  = {
     "name":"Conservation Practice",
-    "value":"Rain Garden",
+    "value":"rain_garden",
```

Comment:
It's pretty great that the tests already check for the labels. Your new stuff got a test for free.